rwslib/rws_requests/odm_adapter.py

```diff
@@ -3,25 +3,44 @@
 
 http://rws-webhelp.s3.amazonaws.com/WebHelp_ENG/solutions/clinical_data_audits/index.html#odm-adapter
```

Comment:
This link doesn't go anywhere anymore

Reply:
Updated to point to Knowledge Hub (along with the others)